docs/get-started/getting-started-databricks.md

```diff
@@ -130,6 +130,8 @@ cluster.
     executor with 1 GPU, which is limiting, especially on the reads and writes from Parquet.  Set
     this to 1/(number of cores per executor) which will allow multiple tasks to run in parallel just
     like the CPU side.  Having the value smaller is fine as well.
+    Note: Please set `spark.task.resource.gpu.amount` default to 1 for single-node Databricks cluster
+    because of Spark local mode.
```

Comment:
officially gpu scheduling isn't supported in local mode, but I'm assuming you tested this and this works?  What if it is just not present?

Reply:
@tgravescs Since the default value is 1 maybe in local mode, we can remove this config to avoid confusion?

Reply:
yeah local mode just say to remove it.

Reply:
Done. @tgravescs